app/subscriber/src/components/basic-search/BasicSearch.tsx

```diff
@@ -13,10 +13,12 @@ export interface IBasicSearchProps {
   onSearch?: (filter: IFilterSettingsModel) => void;
   /** whether to display the header variant of the search */
   inHeader?: boolean;
+  /** whether or not this is the variant that is to be displayed on mobile */
+  mobile?: boolean;
 }
 
 /** Basic search functionality (just search term), and an option to get to the advanced filter */
-export const BasicSearch = ({ onSearch, inHeader }: IBasicSearchProps) => {
+export const BasicSearch = ({ onSearch, inHeader, mobile }: IBasicSearchProps) => {
```

Comment:
How does this new property `mobile` work?  What controls whether it should be mobile or not?  Why not just create a responsive component?

Reply:
oops thought i changed these all over but i missed one - updating now

Reply:
@Fosol - updated